sceptre/cli/__init__.py

```diff
@@ -40,18 +40,18 @@
 @click.option(
     "--var", multiple=True, help="A variable to template into config files.")
 @click.option(
-    "--var-file", type=click.File("rb"),
+    "--var-file", multiple=True, type=click.File("rb"),
     help="A YAML file of variables to template into config files.")
 @click.pass_context
 @catch_exceptions
 def cli(
         ctx, debug, directory, no_colour, output, var, var_file
-):  # pragma: no cover
```

Comment:
why did you remove `# pragma: no cover `?

Reply:
The `cli` function now has full coverage. I thought it would be okay to include it in the coverage report now. That way we will be able to detect if untested paths are added or tests are changed.